Tests/SwiftDocCTests/Test Bundles/ObjCFrameworkWithInvalidLink.docc/symbol-graphs/clang/HelloWorldFramework.symbols.json

```diff
@@ -67,12 +67,12 @@
           {
             "range": {
               "end": {
-                "character": 52,
-                "line": 10
+                "character": 51,
+                "line": 9
               },
               "start": {
-                "character": 5,
-                "line": 10
+                "character": 4,
```

Comment:
Did you edit this symbol graph manually or did you generate it from some Objective-C source code?

Reply:
Manually. I don't know if a source for this test content exists (anymore).